src/ast/irbuilderbpf.h

```diff
@@ -40,6 +40,7 @@ class IRBuilderBPF : public IRBuilder<>
   CallInst   *CreateBpfPseudoCall(Map &map);
   Value      *CreateMapLookupElem(Map &map, AllocaInst *key);
   void        CreateMapUpdateElem(Map &map, AllocaInst *key, Value *val);
+  Value      *CreateMapLookupElem(int mapfd, AllocaInst *key, SizedType &type);
```

Comment:
Nitpick, but could you move this declaration up a line to be next to the other CreateMapLookupElem? The APIs in some of the other files are getting out of control and I don't want that to happen here too - grouping will hopefully make it clearer what's available